sources/reference/admiral-aws-marketplace.md

```diff
@@ -7,100 +7,120 @@ page_keywords: install, microservices, Continuous Integration, Continuous Deploy
+    application
   - The callback URL should be `http://1.2.3.4:50001/auth/github`
-  - Save and copy the keys for this application into Github auth section of
-    Shippable admin panel
+  - Save and copy the keys for this application into GitHub auth section of
+    Shippable Admin panel
 
-- In the `Service Addresses` change the `API` IP address to the public IP
-  of server. The port should remain the same. For our example, the value will
-  be `http://1.2.3.4:50000`
+- In the `Service Addresses` section, change the `API` IP address to the public
+  IP of server. The port should remain the same. For our example, the value
+  will be `http://1.2.3.4:50000`
 
-- In the `Service Addresses` change the `WWW` IP address to the public IP
-  of server. The port should remain the same. For our example, the value will
-  be `http://1.2.3.4:50001`
+- In the `Service Addresses` section, change the `WWW` IP address to the public
+  IP of server. The port should remain the same. For our example, the value
+  will be `http://1.2.3.4:50001`
 
 - Click `Install`. This should set up Shippable services and bring up the UI.
-  Once complete, you should be able to navigate to `http://1.2.3.4:50001` and
-  log into Shippable using a Github account.
-
+  Once complete, navigate to `http://1.2.3.4:50001` and log into Shippable 
+  using a GitHub account.
 
-### Setup superuser account
+### Add-ons
 
-Shippable UI also provides an admin panel which helps users to
+In addition to the Shippable Server Admin panel, within the product UI, 
+Shippable also provides an Admin panel to:
 
-- monitor builds across the system
-- manage build nodes for either the system or individual subscriptions
-- manage user accounts across the system
-- view and manage build, project and subscription
+- Monitor builds across the system
+- Manage build nodes for either the system or individual subscriptions
+- Manage user accounts across the system
+- View and manage builds, projects and subscriptions
 - Perform other housekeeping tasks
 
-Only the accounts that have been added to superuser list will be able to access
-this panel.
+To access it, you'll specify users to be SuperUsers in the `Add-ons` section. 
+Only the accounts that have been added to the SuperUser list will be able to
+access this panel.
+
+- After successfully logging into the Shippable UI using GitHub (or the 
+  configured auth provider), navigate to the Accounts page via the gear icon in 
+  the upper right and copy your account ID
 
-- After successfully logging in using GitHub(or the configured auth provider), navigate to the accounts page and copy the account ID
+- Navigate back to Shippable Server Admin panel `Add-ons` section and scroll 
+  down to `Manage System SuperUsers` sub-section
+    - Paste the account ID from the Accounts page into the text box and hit 
+    `Add`. The account associated with the ID will now have SuperUser 
+    permissions
```

Comment:
Missing punctuation.  Although some of these bullet points have punctuation and some don't.